ghost/core/core/server/data/migrations/versions/5.17/2022-09-27-13-53-remove-click-tracking-tables.js

```diff
@@ -0,0 +1,3 @@
+const {dropTables} = require('../../utils');
+
+module.exports = dropTables(['link_redirects', 'members_link_click_events']);
```

Comment:
This produces an irreversible migration, which is only something we do in majors. We'll have to switch to some other utils, or only use the `.up` from this

Reply:
Aha okay!